src/models/userModel.js

```diff
@@ -4,11 +4,11 @@ const UserSchema = new mongoose.Schema(
   {
```

Comment:
Can you confirm with Hope about the max characters for all the properties that have that property?